openmls/src/lib.rs

```diff
@@ -221,5 +223,8 @@ mod key_store;
 mod tree;
 mod treesync;
 
+// Re-export items from private modules.
+pub use treesync::errors::{ApplyUpdatePathError, PublicTreeError};
```

Comment:
I think this should be re-exported in `group::mls_group::errors` instead. We already re-export other errors there, and I think it makes more sense to do it in the context in which they are needed.

Reply:
You moved it to `error.rs` instead of `group::mls_group::errors.rs` where the other re-exports are. I think it would be good to have all of the sub-types there:

https://github.com/openmls/openmls/blob/main/openmls/src/group/mls_group/errors.rs#L9-L13

Reply:
The treesync module is not part of the mls_group module. So that doesn't really make sense.

Reply:
Let's decide on a strategy later, since this is a bigger issue in other places as well